Hippo.Core/Domain/User.cs

```diff
@@ -32,5 +32,21 @@ public class User
 
         [StringLength(20)]
         public string Kerberos { get; set; }
+
+        [JsonIgnore]
+        public List<Account> Accounts { get; set; }
+
+        internal static void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<User>().HasIndex(a => a.Kerberos);

```

Comment:
just mainly curiosity, but what's the point of indexing kerb?  Are we planning to search on it frequently or anywhere?  AFAIK we kinda just ignore it these days.

Reply:
Harvest had it, just based it off that. I'll remove it and do another migration.